bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -36,7 +36,7 @@ trait Asterism {
     * only possibilities.
     * TODO: should we Church encode this instead?
     */
-  def targets: Target \/ (SiderealTarget, SiderealTarget)
+  def targets: Target \/ (Target, Target)
```

Comment:
There's nothing preventing us from constructing a single asterism that contains a mixture of TOO, sidereal, and non-sidereal targets; but we have a lot of code that asks "is this a sidereal observation?" and I don't know how to answer that question in the general case. We may need to examine every call site and try to figure out why they're asking.